pkg/kubelet/cm/topologymanager/scope.go

```diff
@@ -21,7 +21,7 @@ import (
 
 	"sync"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Can we drop this?

Reply:
yep, same as https://github.com/kubernetes/kubernetes/pull/101432#discussion_r624933346

Reply:
Looks like this (and similar below) out-of-scope changes are still part of the PR?

Reply:
yes, because the overzealous `gopls` on behalf of `vim-go` keeps adding them. Will have a remove pass. Sorry for the churn.